tests/utils/helpers.py

```diff
@@ -40,20 +40,6 @@ def extract_memory_from_execute(result):
     return mem
 
 
-def extract_stack_from_execute(result):
```

Comment:
why do we delete this

Reply:
not needed anymore, see the `test_kakarot.py`, we have `result.stack_values` directly